Assignment/solution_student/css/style.css

```diff
@@ -0,0 +1,118 @@
+
+h1 {
+	text-align: center;
+	line-height: 0.5;
+	font-size: 20px;
+	font-family: arial;
+	padding-bottom: 8px;
+}
+
+h2 {
+	text-align: center;
+	font-size: 35px;
+	line-height: 0.05;
+	font-family: arial;
+	padding-bottom: 30px;
+}
+
+h3 {
+	text-align: left;
+	margin-left: 10px;
+	font-size: 20px;
+	line-height: 0.05;
+	font-family: arial;
+	padding-bottom: 15px;
+
+}
+
+h4 {
+	text-align: left;
+	margin-left: 10px;
+	font-size: 15px;
+	font-weight: 500;
+	line-height: 0.05;
+	font-family: verdana;
+	padding-bottom: 15px;
+
+}
+
+/*i'm not entirely sure how to separate out pieces
```

Comment:
Good question — you were really close in your markup by wrapping that particular part in an <em>, which can be targeted and style specifically as different from the rest of the stuff in the same element